plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -84,7 +84,11 @@ export async function createRouter(
   const taskBroker = new StorageTaskBroker(databaseTaskStore, logger);
   const actionRegistry = new TemplateActionRegistry();
   const workers = [];
-  for (let i = 0; i < (taskWorkers || 1); i++) {
+  for (
+    let i = 0;
+    i < (typeof taskWorkers === 'undefined' ? 1 : taskWorkers);
```

Comment:
Could make this `taskWorkers ?? 1`

Reply:
Thanks :) sounds like it'll work better.

Reply:
done